src/shared/components/dateRangePicker/NewDatePicker.tsx

```diff
@@ -129,38 +140,16 @@ const DatePickerMenu: FC<Props> = ({onCollapse, timeRange, timeRangeLabel}) => {
       }
     }
     setDateRange([start, end])
-    let startInput = start
-    let endInput = end
```

Comment:
we should not be setting this to a Date, and latter on overwriting with a string.